packages/app-project/src/components/ZooHeaderWrapper/ZooHeaderWrapperContainer.js

```diff
@@ -10,8 +10,11 @@ function useStore() {
   return { user }
 }
 
-export function signOut(user) {
+export async function signOut(user) {
+  await auth.signOut()
```

Comment:
What happens here if there’s an API error? Do I stay logged in on the page but logged out in Panoptes?

Reply:
This might need to be wrapped in a try/catch block, with some error handling code for the case where the API errors or maybe times out.

Reply:
Yes definitely could use some error handling.